src/test_data/text.txt

```diff
@@ -0,0 +1,3 @@
+Hello,
```

Comment:
wat is this for ?

Reply:
This folder is used for test.
The way go is working in test mod is for a mystery. I tried to put the absolute path to the file. It did not work the only thing that work is this way to test the function that open file and read it.